dataplicity/m2mmanager.py

```diff
@@ -153,7 +153,9 @@ def on_instruction(self, sender, data):
         elif action == 'open-portredirect':
             device_port = data['device_port']
             m2m_port = data['m2m_port']
-            self.client.port_forward.redirect_port(device_port, m2m_port)
+            self.client.port_forward.redirect_port(
```

Comment:
Why keyword arguments, when there are only two essential parameters?

Reply:
I believe the reason was that they were in reversed order. i.e. in the function being called the first parameter was actually m2m port. I had the call without named parameters and I had mixed the order. Because both of these parameters were ints, I thought that it would be nice to state them verbosely so that there wouldn't be any confusion to which one is which. Of course I can reverse the order if you want

Reply:
I'd probably make them consistent throughout, rather than clarify with kwargs. That kind of thing will only come back to bite you.